sphinx/ext/graphviz.py

```diff
@@ -77,7 +78,8 @@ def run(self):
                     'Graphviz directive cannot have both content and '
                     'a filename argument', line=self.lineno)]
             env = self.state.document.settings.env
-            rel_filename, filename = env.relfn2path(self.arguments[0])
+            argument = get_image_filename_for_locale(self.arguments[0], env)
```

Comment:
Try to substitute to `filename.{language}` (by default `figure_language_filename`).
We still keep on supporting them.